### ISSUE TITLE
Script to download and copy files into docker container 

### ISSUE THREAD

**Issue by fausto-svds** [jeff-svds/query-engine-comparison#1]
1) download files 
2) unzip into /data dir
3) mount /data  dir on container as volume 
4) put into hdfs 
5) point table creation at relevant hdfs dir

**Comment by jeff-svds** [comment]
Tried to use `curl` and `wget` to download the data, but couldn't get it to work. I think the AWS S3 security mechanisms are preventing downloads from programs other than web browsers. For now, the setup script assumes that the gzipped tar file is in the user's `/data` directory and will take it from there. See latest commit https://github.com/jeff-svds/query-engine-comparison/commit/e68d5f6078c6bd5018da26f5cff13c80eac80392

**Comment by fausto-svds** [comment]
@jeff-svds also they can just download and move it into /data as well right? its only 200 MB

**Comment by jeff-svds** [comment]
@fausto-svds Yeah that can be done. But, again, their wifi is overloaded and you may save everyone time by having the thumb drive option.
